src/GitVersionCore/Configuration/Config.cs

```diff
@@ -26,6 +26,9 @@ public Config()
         [YamlMember(Alias = "assembly-informational-format")]
         public string AssemblyInformationalFormat { get; set; }
 
+        [YamlMember(Alias = "assembly-file-versioning-format")]
+        public string AssemblyFileVersioningFormat { get; set; }
```

Comment:
Shouldn't we provide `AssemblyVersioningFormat` as well?

Reply:
Addressed in the latest patch with some code cleanup